core_backend/app/llm_call/llm_prompts.py

```diff
@@ -221,21 +226,24 @@ def get_prompt(cls) -> str:
 """
     + f"""
 For "answer", understand the extracted information and user question, solve the \
-question step by step, and provide the answer in the language of the question. \
+question step by step, and then provide the answer. \
 If no useful information was found in REFERENCE TEXT, respond with \
-"{ANSWER_FAILURE_MESSAGE}"
+"{RAG_FAILURE_MESSAGE}".
 """
     + """
-Example responses:
+EXAMPLE RESPONSES:
 {{"extracted_info": ["Pineapples are a blend of pinecones and apples.", "Pineapples \
 have the shape of a pinecone."], "answer": "The 'pine-' from pineapples likely come \
-from the fact that pineapples are a hybrid of pinecones and apples and its pinecone- \
+from the fact that pineapples are a hybrid of pinecones and apples and its pinecone\
 -like shape."}}
 {{"extracted_info": [], "answer": "FAILED"}}
 
+IMPORTANT NOTES ON THE "answer" FIELD:
+- Answer in the language of the question.
+- Do not include any information that is not present in the REFERENCE TEXT.
+
 REFERENCE TEXT:
-{context}
-""".strip()
```

Comment:
I remember you saying this was causing issues. Remind me what that was?

Reply:
Oh I think it was the text dedent function. Got rid of Strip because it wasn't necessary ( I tested the formatting with print statements)!